src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -141,12 +142,13 @@ class LineExecutionProcessPool:
 
     def __init__(
         self,
-        flow_executor: FlowExecutor,
+        flow_executor: Union[FlowExecutor, ScriptExecutor],
```

Comment:
ScriptExecutor is a subclass so we don't need Union here.

Reply:
Removed.